content/about.md

```diff
@@ -2,4 +2,21 @@
 title: "About"
 ---
 
-**A Fistful of Links** is a weekly newsletter about leadership, technology, books, and anything else we felt compelled to share with others, brought to you by Og Maciel and Mirek Długosz.
+**A Fistful of Links** is a weekly newsletter about leadership, technology, books, and anything else we felt compelled to share with others, brought to you by **The Editors**.
```

Comment:
Since "Editors" are right below, would it make sense to write:

```
...brought to you by:
Og Maciel       Mirek Długosz
info on Og      info on Mirek
```
?